src/routes/+page.svelte

```diff
@@ -59,10 +61,10 @@
 	let sortDirection = $state<SortDirection>(SORT_DIRECTIONS[0]);
 
 	const userDimensions = $state<UserDimensions>({
-		length: 0,
-		width: 0,
-		height: 0,
-		unit: 'cm'
+		length: Number($page.url.searchParams.get('length')) ?? 0,
```

Comment:
Would be great if dimensions will be also cleared from url if changed in bag input

Reply:
Sorry, I did not get it. Could you explain again? 😅

Reply:
No worries! What I meant is this scenario:
1. User click on shared link with dimensions set
2. The UI handle this, setting all values from url to input.
3. User changed the bag dimensions in input fields or even reset them.
4. Shared bag dimensions are also clear, so when for example user refresh page, old shared values were not shown again.